README.md

```diff
@@ -63,3 +67,9 @@ To configure sentry, create a new project at https://sentry.wnyc.org/sentry/.
 The creation of that project will automatically generate a DSN. It will look something like this: `https://<KEY>@sentry.wnyc.org/<PROJECT_ID>`.
  
 In the project that imports this library, make sure to include the `sentryDSN` parameter when initializing the fastboot server. Best practices around this are to set `sentryDSN` to an environment variable which can be configured at deployment time.
+
+## Statsd
+Statsd is a metrics collection and aggregation agent that can plug into a number of backends. We use it to send application metrics to our graphite backend. The collector is installed on the same machine that graphite runs on, and the client responsible for generating and sending metrics is included as a middleware here. The source code can be found here: https://github.com/uber/express-statsd
+
+The `env` and `serviceName` parameters passed into the fastboot constructor are used to namespace the metrics being sent to statsd.
```

Comment:
suggested values would be a good thing to include here as well

Reply:
sounds good, I'll add them.